roles/openshift_health_checker/openshift_checks/package_version.py

```diff
@@ -10,11 +10,9 @@ class PackageVersion(NotContainerizedMixin, OpenShiftCheck):
     tags = ["preflight"]
 
     def run(self, tmp, task_vars):
-        rpm_prefix = get_var(task_vars, "openshift", "common", "service_type")
-        openshift_release = get_var(task_vars, "openshift_release")
-
         args = {
-            "prefix": rpm_prefix,
-            "version": openshift_release,
+            "prefix": get_var(task_vars, "openshift", "common", "service_type"),
+            # this normalizes openshift_release (from the inventory, or defaults in openshift_facts):
```

Comment:
Unnecessary comment?
- The `normalized_release` name tells a good part of it (should we name it `normalized_openshift_release` instead? So that it is clear this is not Ansible's version or anything else?).
- The `openshift_release` can be set from a good number of places, the many ways to set variables in Ansible. I'd avoid suggesting where it was defined. Defer it to Ansible's doc.
  Or, if we really want to assist whoever is maintaining this code, then perhaps that would belong in a more central place, like a "developers guide" to health checks?